docs/commands/super.md

```diff
@@ -659,20 +659,32 @@ the `super` command, but it turns out that `super` can hold its own when
 compared to other analytics systems.
 
 To illustrate comparative performance, we'll present some informal performance
-measurements among `super`,
-[`DuckDB`](https://duckdb.org/),
-[`ClickHouse`](https://clickhouse.com/), and
-[`DataFusion`](https://datafusion.apache.org/).
+measurements among SuperDB,
+[DuckDB](https://duckdb.org/),
+[ClickHouse](https://clickhouse.com/), and
+[DataFusion](https://datafusion.apache.org/).
```

Comment:
I try to stick to the backtick-enclosed fixed width font when showing things as typed by a user. We reference all these by their command names further down, but I figured where we're hyperlinking to them up top we can use their real names here.